javascript/implementation/GinkInstance.ts

```diff
@@ -281,6 +281,7 @@ export class GinkInstance {
     private receiveCommit(commitBytes: BundleBytes, fromConnectionId?: number): Promise<BundleInfo> {
         return this.store.addBundle(commitBytes).then((bundleInfo) => {
             this.logger(`commit from ${fromConnectionId}: ${JSON.stringify(bundleInfo)}`);
+            this.iHave.markAsHaving(bundleInfo);
```

Comment:
This solved the problem of the second peer getting an error. When the instance loops through and sends to its peers, it never called markAsHaving() on its own commit.